### PR TITLE
Closes #6 - test: add unit tests for 100% coverage on the after validator

### LINKED ISSUE #6: Add unit tests to the *after* validator for 100% coverage
Use the storybook and the `notNull` validator's tests as a guide, and add tests that mirror the stories.

Acceptance criteria:
* 100% test coverage of the validator.
* Style, etc. the same as the other unit tests.

### PR DESCRIPTION
**Pull request by Lithowan**

### CODE REVIEW

File: src/baseValidators/after/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Left, Nothing, parseDate, Right } from 'sanctuary'
+
+import { NOT_AFTER_DATE_TIME } from '../../errorTypes'
+import after from './'
+
+const testDate = parseDate('2001-01-01T00:00:00.000Z')
+
+describe('baseValidators:after', () => {
```
Comment (chasm) [inline src/baseValidators/after/index.spec.js]: This isn't a bad approach, but is not consistent with the other tests. If you make it so, we can merge it today.

Comment (Lithowan) [reply src/baseValidators/after/index.spec.js]: Which aspect of that part do you mean, using a variable or something else?

Comment (Lithowan) [reply src/baseValidators/after/index.spec.js]: I think I figured out what you mean and I've updated the tests. Let me know if I missed something or if there's anything else.

Comment (chasm) [reply src/baseValidators/after/index.spec.js]: Yep. Good job. We want the tests, when run, to read the same to reduce cognitive load. That's the theory, anyway. ;-)

**Pull request merged**